js/panel_controller.js

```diff
@@ -19,11 +19,19 @@
 
     window.drawAttention = function() {
         if (inboxOpened && !inboxFocused) {
+            if( window.keepClear ) {
+                clearInterval( window.keepClear );
+                delete window.keeClear;
+            }
             extension.windows.drawAttention(inboxWindowId);
         }
     };
     window.clearAttention = function() {
-        extension.windows.clearAttention(inboxWindowId);
+        if( window.keepClear ) {
+            clearInterval( window.keepClear );
+            delete window.keeClear;
```

Comment:
It seems clear that it should, yes.  Sorry I did not notice this.  8(

Should I fix and re-request pull?